man/sphinx/conf.py

```diff
@@ -161,7 +161,7 @@
 # html_file_suffix = None
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = "grass81Documentationdoc"
+htmlhelp_basename = "grass83Documentationdoc"
```

Comment:
These, too, look like good candidates for switch to version-less names, but perhaps for a different commit with some testing if these pieces even work.

Reply:
I'd like to keep the version in the page titles, otherwise we'll have (SEO) confusion...